code/script.js

```diff
@@ -262,82 +322,146 @@ const selectQuestion = () => {
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
 
-  if (category === 'hair color') {
+const value = questions.value
+ if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
+      value: value,
```

Comment:
when you are passing as the value variable with the same name as a key you do not need to specify it:) 
``` 
{ 
  category: category,
  name: name
}
```
can be written `{category, name}`